dxp/javascript_minification.md

```diff
@@ -0,0 +1,86 @@
+# JavaScript Minification in DXP
+
+This article describes DXP infrastructure to minify JavaScript source files. It is related to [Resource Serving in DXP](resource_serving.md).
+
+## MinifierUtil, JavaScriptMinifier and frontend-js-minifier
+
+These three pieces conform the basic infrastructure available in DXP to minify JavaScript code.
+
+-   [JavaScriptMinifier](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portal/minifier/JavaScriptMinifier.java): describes the interface that services to minify JavaScript must implement.
+
+-   [MinifierUtil](https://github.com/liferay/liferay-portal/blob/master/portal-impl/src/com/liferay/portal/minifier/MinifierUtil.java): provides a way to retrieve the `JavaScriptMinifier` with highest service ranking currently deployed to DXP. Note that you can also obtain a reference to a `JavaScriptMinifier` (or all of them) using standard OSGi mechanisms.
```

Comment:
Backticks around `MinifierUtil` please.

> using standard OSGi mechanisms

What are those mechanisms? I have no idea. Could you turn that into a link somewhere informative?